lib/find.js

```diff
@@ -1,13 +1,12 @@
-function find(selector, el) {
-  let element;
-
-  if (el) {
-    element = el;
-  } else {
-    element = document.body;
+function find(selector, context = null) {
+  let els = [];
+  if (selector instanceof HTMLElement) {
+    els.push(selector);
+  } else if (typeof selector === 'string') {
+    const startElement = (context) ? find(context) : [document.body];
```

Comment:
It's an array because when there is a context provided, `find()` call itself to get the context element. `find()` always returns an array. Eventually the method should be able to handle multiple elements for the context.